### ISSUE TITLE
Songs are inconsistent in end JSON.

### ISSUE THREAD

**Issue by ljharnish** [serpapi/google-search-results-nodejs#15]
After getting results, in `knowledge_graph`, sometimes a songs artist will come up as `artis` and not `artist`.
![image](https://user-images.githubusercontent.com/76492103/221089019-7df61fa6-195b-442a-8a81-126f47e9da65.png)

Also, when getting a song from `answer_box`, it will sometimes mix up all the data like this: 
![image](https://user-images.githubusercontent.com/76492103/221088851-55697eeb-9150-45f8-b7a9-a9e6b7a355f4.png)
That video link also was not there the last time I ran this.
Please fix these if you can.

**Comment by ljharnish** [comment]
Just figured this new info out:

For some reason, even though I set my location to the US, it is giving me Indonesian results when searching for Lucid Dreams. This explains the `artis` and `dengarkan`. The `answer_box` is still inconsistent though.

**Comment by dimitryzub** [comment]
@ljharnish thank you for opening this issue.

- Could you clarify which engine are you using? From what I understand it seems to be a Google Search API. 
-What exactly are you trying to get from the results?
- What is the search query?
- Could you give us an inspect link so we can take a look? https://serpapi.com/searches

![image](https://user-images.githubusercontent.com/78694043/221102732-20af070f-6b82-40d6-a75d-a1aa7c8f0ca2.png)

**Comment by ljharnish** [comment]
I'm using the Google Search API, yes.
I'm trying to get the song data from the answer to show it on my site.
Here's the inspect link: https://serpapi.com/searches/63f83c959bddf791fe648b8d/inspect
Looking at that link myself, it looks correct but on my site it is still incorrect.

**Comment by dimitryzub** [comment]
@ljharnish Thank you 👍

> but on my site it is still incorrect

Could you explain what you meant by saying "incorrect"?

**Comment by ljharnish** [comment]
@dimitryzub On my site, the values in `answer_box` are still scrambled into other keys. Even though the values are uncrambled in the inpect page.

**Comment by dimitryzub** [comment]
> @dimitryzub On my site, the values in `answer_box` are still scrambled into other keys. Even though the values are uncrambled in the inpect page.

@ljharnish could you please have a look at the illustrations below? Did you have any misunderstandings with this? 

![image](https://user-images.githubusercontent.com/78694043/222368063-2a59ec06-f0fa-4415-a018-35ad6509861e.png)


![image](https://user-images.githubusercontent.com/78694043/222368091-780f72c1-ba81-442d-82d0-135298421b10.png)

@kagermanov27 @tanys123 could you guys have a look whenever you can? Is it as how it is supposed to be? [Inspect page](https://serpapi.com/searches/6400564324e88ec7a9f9e1a0/inspect).

If it's not as supposed to be, we can create an issue on public-roadmap.

**Comment by ljharnish** [comment]
@dimitryzub Yes, I uderstand those. The data is correct, its just not in the correct place: ![DevTools Screenshot](https://user-images.githubusercontent.com/76492103/221088851-55697eeb-9150-45f8-b7a9-a9e6b7a355f4.png)

The `album` value has the data from the `released`, the `artist` has the data from the actual `album`, and the `released` has the entire `artist` in it.

**Comment by ilyazub** [comment]
@ljharnish thanks for your feedback. :+1: I'm sorry for the late reply.

Please use the correct data from the `knowledge_graph.album` and `knowledge_graph.released`.
![image](https://github.com/serpapi/public-roadmap/assets/282605/aa5bcaf6-c913-42e9-bd2a-ef3573c8a89a)

The `answer_box.album` and `answer_box.released` are incorrectly parsed indeed. I created an issue in our public roadmap: https://github.com/serpapi/public-roadmap/issues/1735

![image](https://github.com/serpapi/google-search-results-nodejs/assets/282605/2d95b607-85f2-4f98-8d67-e07c800ceaef)
(ref: https://serpapi.com/playground?q=XXXKIRBY)
